server/src/main/java/io/deephaven/server/runner/Main.java

```diff
@@ -62,6 +63,10 @@ public static Configuration init(String[] args, Class<?> mainClass) throws IOExc
 
         final Configuration config = Configuration.getInstance();
 
+        // After logging and config are working, redirect any future JUL logging to SLF4J
```

Comment:
It seems like there should be a saner place to put this, but we don't have a centralized place to set up logging for all processes, so I started just with the two server mains. The initialize method in the log factory provider doesn't work, as that won't be called in this case until after the error in this bug has happened.

I'm not totally convinced we should have a single place for this, since the various client CLI mains seem to prefer to send to logback, so we wouldn't want to necessarily do the same log setup that exists in this class anyway?

Open to feedback and advice.

Reply:
I'm happy to be explicit here for now, generalize later if necessary.